roles/openshift_facts/library/openshift_facts.py

```diff
@@ -730,6 +730,9 @@ def set_manageiq_facts_if_unset(facts):
             raise OpenShiftFactsInternalError(
                 "Invalid invocation: The required facts are not set"
             )
+    if 'use_manageiq' in facts['common']:
+        if facts['common']['use_manageiq'] is not None:
+            return facts
```

Comment:
So...  any facts passed in to the openshift_facts module with a value of None are actually discarded. The main purpose was to give us a way to conditionally set the fact based on the presence/lack of presence of the variable.

https://github.com/brenton/openshift-ansible/blob/manageiq/roles/openshift_facts/library/openshift_facts.py#L1286-L1293 is where we discard items that are set to None or to an empty string.